server.js

```diff
@@ -1,33 +1,48 @@
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
+import netflixData from './data/netflix-titles.json'
+
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+
 app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
-// Start the server
+
+app.get('/all', (req, res) => {
+  res.json(netflixData)
+})
+
+app.get('/shows/', (req, res) => {
+  const shows = netflixData.filter((item) => item.type === "TV Show")
+  res.json(shows)
+})
+
+app.get('/shows/:id', (req, res) => {
+  const id = req.params.id
+  const showsId = netflixData.filter((shows) => shows.type === "TV Show" && shows.show_id === +id)
+  res.json(showsId)
+})
+
+app.get('/year/:year', (req, res) => {
```

Comment:
I think, following REST, the `/shows`,  `/movies` and `/year/:year` endpoint should all be merged into one, and make use of query params to filter

* `/shows` - returns everything
* `/shows?type=tv` - returns only tv shows
* `/shows?type=movie` - returns only movies
* `/shows?year=2019` - returns anything from 2019

This way you could chain things together without having to duplicate lots of logic. Like 'show me movies from 2019' = `/shows?type=movie&year=2019`